SocialCareCaseViewerApi/V1/UseCase/CaseStatusesUseCase.cs

```diff
@@ -30,5 +31,29 @@ public ListCaseStatusesResponse ExecuteGet(long personId)
 
             return response;
         }
+
+        public void ExecutePost(CreateCaseStatusRequest request)
+        {
+            var persons = _databaseGateway.GetPersonByMosaicId(request.PersonId);
+            if (persons == null) throw new PersonNotFoundException($"'personId' with '{request.PersonId}' was not found.");
+
+            var type = _databaseGateway.GetCaseStatusTypeWithFields(request.Type);
+            var typeDoesNotExist = type == null;
+            if (typeDoesNotExist) throw new CaseStatusTypeNotFoundException($"'type' with '{request.Type}' was not found.");
+
+            request.TypeId = type.Id;
```

Comment:
Instead of mutating the request I'd prefer if we pass this value into .CreateCaseStatus

Reply:
This was actually not necessary, removed